contrib/font2psf

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
You are right, it is actually removal of a Bash specific code. Feel free to edit.

There are people in the Open Source community trying to reach the goal of a GNU/Linux distribution independent on Bash as much as possible. They target to scripts, where use of bash specific code does not bring any benefit, and it is easy to remove bashisms.

These people are inspecting code in the whole GNU/Linux distribution (openSUSE, maybe others) and sending patches to many projects. They do not fix other bugs in these scripts. We are accepting most of their patches.

Reply:
To be honest, I don’t see any benefit in such patches, especially in the contrib/ directory. Otherwise I would have already done it. 

If you see benefit in removing bash-specific code, then redo the patch and rewrite all the scripts in this directory, or remove this patch.

Reply:
Actually, rc/suse-kbd.rc also affected by the patch is a ~30 years old piece of code, totally out of date and no more used for ~10 years (since migration to systemd). It would be better to drop it.